src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Common/Microsoft.AspNetCore.Razor.LanguageServer.Common.csproj

```diff
@@ -7,6 +7,10 @@
     <IsShippingPackage>false</IsShippingPackage>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.VisualStudio.Telemetry" Version="$(MicrosoftVisualStudioTelemetryVersion)" />
```

Comment:
Why do we need this reference now (or I suppose I'm actually more surprised we didn't need it before)?

Reply:
We did have the reference before, I just accidentally removed it when resolving merge conflicts, so this is putting it back https://github.com/dotnet/razor-tooling/blob/main/src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Common/Microsoft.AspNetCore.Razor.LanguageServer.Common.csproj#L14